### DISCUSSION TITLE
dotplot/synteny track

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse-components#3550] [Q&A] [answered]
Hello,
I tried reproducing the example of grape vs peach given in the quickstart guide "web setup using CLI" but i ran into some trouble with it.
I created a new version of jbrowse2, downloaded the grape.fa and peach.fa created the .paf file and ran into some trouble loading it through the cli so i modified directly the config file which seemed to be working for the dotplot
![image](https://user-images.githubusercontent.com/91090403/222195291-dc69e9c3-ec85-4fa9-9cc1-f9080c908398.png)
But when i tried opening a linear synteny view it kept loading forever
![image](https://user-images.githubusercontent.com/91090403/222197784-643fc1a8-717b-4eae-a3de-4e1398a7f303.png)
Here is the config for reference
![image](https://user-images.githubusercontent.com/91090403/222194661-6f054c39-e267-4b31-a2a2-8466b9a2b14f.png)

Where did i go wrong ?

**Comment by cmdcolin** [comment, marked as answer]
There was a bug that was recently fixed where the code could get into an infinite loading state, it was fixed and is pending release https://github.com/GMOD/jbrowse-components/pull/3556

You can fix it on your current version by just showing a larger region probably, the infinite loading state is because it just doesn't have any features being drawn
